src/OpenSage.FileFormats.Refpack/OpenSage.FileFormats.RefPack.csproj

```diff
@@ -0,0 +1,7 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Sorry to be a pain, but please could you use consistent casing for the project (and folder) name? RefPack, not Refpack.

Reply:
Now there seem to be duplicate files, under both Refpack and RefPack...

Reply:
Case insensitivity on windows is really disgusting....

Reply:
This seems to be correct for me. There are no multiple files on git